README.md

```diff
@@ -92,6 +92,12 @@ props details:
           <td></td>
           <td>function returning html node which will act as notification container</td>
         </tr>
+        <tr>
+          <td>isLastTop</td>
```

Comment:
Need a better name.

Reply:
`isReverse` ?

Or use `isAddOnTop`, mean is add a new notification to top or bottom and it not a global change yet.

Reply:
`newestOnTop`